packages/mobile/src/screens/user-list-screen/UserListItem.tsx

```diff
@@ -63,11 +66,12 @@ const useStyles = makeStyles(({ spacing, palette }) => ({
 }))
 
 type UserListItemProps = {
+  tag: string
```

Comment:
nit: made the same comment in @sddioulde 's PR - `tag` feels overloaded as a term, is there something else we could use? Also, should this maybe be an enum?

Reply:
yes to all the above, ideally we have a better prop name, and then also define these tags in a common file. im down to look into that moving forward, just don't want to slow down this work for that large refactor? https://linear.app/audius/issue/C-541/consolidate-user-list-tags-and-rename-props